demo/KevinDemo/Resources/en.lproj/Localizable.strings

```diff
@@ -48,20 +48,20 @@
 "country_name_is" = "Iceland";
 "country_name_li" = "Liechtenstein";
 
-"details_section_title" = "Enter details";
-"email" = "E-mail";
-"amount" = "Payment amount";
-"currency_eur" = "EUR";
-"agreement_checkmark" = "I have read and agree with %@ and %@";
-"terms_and_conditions" = "Terms & Conditions";
-"privacy_policy" = "Privacy Policy";
+"window_main_details_lable" = "Enter details";
+"window_main_email_label" = "E-mail";
+"window_main_amount_label" = "Payment amount";
+"window_main_amount_field_currency" = "EUR";
+"window_main_terms_privacy_policy" = "I have read and agree with %@ and %@";
+"window_main_terms_privacy_policy_clickable_terms" = "Terms & Conditions";
+"window_main_terms_privacy_policy_clickable_policy" = "Privacy Policy";
 
-"donate_button_title" = "Donate • %@ %@";
+"window_main_proceed_button" = "Donate • %@ %@";
 
 "error_alert_title" = "Oops!";
 "error_alert_description" = "Something went wrong! Please debug the code for more details..";
 "payment_was_canceled" = "Payment was canceled by the user!";
 "success_alert_title" = "Success!";
 "success_alert_description" = "Donation was successful";
```

Comment:
I renamed localisation keys to be the same with the ones in Localise, but could not find keys for this keys.